Source/Extensions/StringExtensions.swift

```diff
@@ -39,12 +39,12 @@ extension String {
         let renderer = UIGraphicsImageRenderer(size: pageRect.size)
         
         return renderer.image {
-            UIColor.white.set()
+            UIColor.clear.set()
```

Comment:
I think that this whole method (`func vectorPDFImage(forClass: AnyClass) -> UIImage?`) was something I objected to from the beginning. Isn't it easier to remove it instead of patching?

Reply:
It can be in another ticket to migrate all images to assets. It may affect SPM resource. We need to make a ticket for it and test for all dependancy management.

Reply:
This is why I wouldn't have liked that to be committed in the first place. We did something wrong the first time around, there were too many things to keep unpicking, and now we're never finding time to put it right, keep kicking the can down the road.

Reply:
@ehab-al-cko can you make a tech investment story ( probably paste the link in here as a reply ) for this which we can bring in later in the sprint. I know we have few concerns till we can manage to track and address them sooner would be nice to see that happening.

Reply:
[Jira Story](https://checkout.atlassian.net/browse/PIMOB-1227?atlOrigin=eyJpIjoiODE1ODljYzNkYWQyNDE5Y2I4YmZkODFkNDBjODE4YWYiLCJwIjoiaiJ9)